.github/CODEOWNERS

```
@@ -1,7 +1,10 @@
 # Each line is a file pattern followed by one or more owners.
 
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence,
-# these owners will be requested for review when someone
-# opens a pull request.
-*       @INLGuy @zenine07
+# These owners will be the default owners for everything in the
+# repo. Unless a later match takes precedence, these owners will be
+# requested for review when someone opens a pull request.
+* @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
```

Should we all be code owners until we figure out how we want to pare down this list?

```suggestion
* @dav3r @felddy @hillaryj @INLGuy @jsf9k @mcdonnnj @zenine07
```